src/Pretzel.Logic/Commands/CommandInfoAttribute.cs

```diff
@@ -1,14 +1,18 @@
 using System;
 using System.Composition;
 
-namespace Pretzel.Commands
+namespace Pretzel.Logic.Commands
 {
     [MetadataAttribute]
     [AttributeUsage(AttributeTargets.Class, AllowMultiple = false)]
     public sealed class CommandInfoAttribute : ExportAttribute
     {
         public string CommandName { get; set; }
```

Comment:
Since we are already breaking (almost) all the things, what do you think of taking the opportunity to get rid of the redundant `Command` part?

Reply:
I'm not sure, do you mean the namespace or the naming in general?

Reply:
The naming of properties for this attribute, like using `Name` instead of `CommandName`.
I think that `[CommandInfo(Name...` is pretty self-explanatory without having to repeat `Command`, but maybe it is just me.

Reply:
Agreed. Removed redundant naming.

Reply:
Thanks!